### PR TITLE
Remove inline styles from Relation.php

### PR DESCRIPTION
**Pull request by mohitjawanjal**
Related to #12262 

Signed-off-by: Mohit Jawanjal <mohit.jawanjal@imnica.com>

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

### CODE REVIEW

File: libraries/classes/Relation.php [hunk 1]
```diff
@@ -101,16 +101,16 @@ public function getRelationsParamDiagnostic(array $cfgRelation)
         $retval = '<br>';
 
         $messages = array();
-        $messages['error'] = '<span style="color:red"><strong>'
+        $messages['error'] = '<span class="span_color_red"><strong>'
```
Comment (mauriciofauth) [inline libraries/classes/Relation.php]: You can use the `caution` class here. See:
https://github.com/phpmyadmin/phpmyadmin/blob/695fe3a994b18ad21a4c1119305e25edd71c7036/themes/pmahomme/css/common.css.php#L1026-L1028

File: libraries/classes/Relation.php [hunk 2]
```diff
@@ -101,16 +101,16 @@ public function getRelationsParamDiagnostic(array $cfgRelation)
...
             . '</strong></span>';
 
-        $messages['ok'] = '<span style="color:green"><strong>'
+        $messages['ok'] = '<span class="span_color_green"><strong>'
             .  _pgettext('Correctly working', 'OK')
             . '</strong></span>';
 
-        $messages['enabled']  = '<span style="color:green">' . __('Enabled') . '</span>';
-        $messages['disabled'] = '<span style="color:red">'   . __('Disabled') . '</span>';
+        $messages['enabled']  = '<span class="span_color_green">' . __('Enabled') . '</span>';
```
[the remark this answers is not in the document]
Comment (mauriciofauth) [reply libraries/classes/Relation.php]: Great! Once you submit the new changes, we can merge this pull request.

**Comment by mohitjawanjal** [comment]
Sorry my git kung-fu is not very good today. :( Going to re-submit it on a different PR. @mauriciofauth Do you think creating a new branch would be a good idea or should I work with master?

File: themes/original/css/common.css.php [hunk 3]
```diff
@@ -3434,4 +3437,4 @@
 /* side menu */
 #name-panel {
     overflow:hidden;
-}
+}
```
Comment (mauriciofauth) [inline themes/original/css/common.css.php]: Could you add the new line at the end of file again?

File: themes/pmahomme/css/common.css.php [hunk 4]
```diff
@@ -3683,4 +3686,4 @@
 /* side menu */
 #name-panel {
     overflow:hidden;
-}
+}
```
Comment (mauriciofauth) [inline themes/pmahomme/css/common.css.php]: Could you add the new line at the end of file again?

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

**Comment by mauriciofauth** [comment]
@mohitjawanjal You can keep the master branch in this PR, but it is recommended to use a separate branch.

File: themes/pmahomme/css/common.css.php [hunk 5]
```diff
@@ -1026,6 +1026,9 @@
 span.caution {
     color: #FF0000;
 }
+span.success {
+    color: green;
```
Comment (mauriciofauth) [inline themes/pmahomme/css/common.css.php]: This line contains trailing whitespace. Could you remove it?

**Comment by mauriciofauth** [comment]
Merged, thanks for your contribution!

**Pull request merged**